tdvt/tdvt/exprtests/regexcalcs/setup.string.regexp.extract.full.txt

```diff
@@ -0,0 +1,20 @@
+REGEXP_EXTRACT([str0], '(E.{3})')
+REGEXP_EXTRACT([key], "(key)") //"key"
+REGEXP_EXTRACT([str0], "(z)") //"Null"
+REGEXP_EXTRACT("toyota", "(o)") //"o"
+REGEXP_EXTRACT([num0], "(0{2})") //"00"
```

Comment:
This is [what I mentioned](https://github.com/tableau/connector-plugin-sdk/pull/969/files?diff=unified&w=1#diff-8f1585626d93006bf876be05662d1acc742716fe6d403dc797da75b978abecb3R48) above. Needs a different test case, looks like.